src/register/Register.jsx

```diff
@@ -36,6 +36,7 @@ export default function Register() {
                 headers: {
                     "Accept": "application/json",
                     "Content-Type": "application/json",
+                    "Access-Control-Allow-Origin": {CORS_ORIGIN}
```

Comment:
Does the browser not automatically add these? I have no clue about front end I always thought the browser would auto generate these.

Reply:
@epicadk , yeah, unfortunately it did not. Before I added this, the api calls received error stating `No Access-Control-Allow-Origin header is present...` and it was cleared after I added this.